src/marketing/utilities/animations.scss

```
@@ -7,3 +7,41 @@
     transform: scale3d(1.025, 1.025, 1.025);
   }
 }
+
+// Animated arrow symbol, used in marketing links, buttons, etc.
+.arrow-target-mktg {
```

```suggestion
.btn-mktg,
.link-mktg,
.arrow-target-mktg {
```

Could we just include these selectors here as well? Since they're always expected to be used with arrows, and otherwise we just make it more difficult to do the right thing 👀